test/cmd/newapp.sh

```diff
@@ -361,6 +361,7 @@ os::cmd::expect_success 'oc delete imagestreamtag ruby:latest'
 os::cmd::expect_failure_and_text 'oc new-app --image-stream ruby https://github.com/openshift/rails-ex --dry-run' 'error: multiple images or templates matched \"ruby\":'
 # when only 2.3 exists, there is a single partial match (2.3)
 os::cmd::expect_success 'oc delete imagestreamtag ruby:2.2'
+os::cmd:;expect_success 'oc delete imagestreamtag ruby:2.4'
```

Comment:
if you want to keep this consistent, you should be deleting 2.2 and 2.3 (so that the only tag left is the 2.4 tag which is also referenced by the "latest" tag).

of course you'll also have to update the next test that expects success with "2.3" to instead expect success with "2.4"

Reply:
updated